_posts/2015-07-20-self-organizing-team-2-footballs.md

```diff
@@ -0,0 +1,85 @@
+
+
+1. Self organization of teams when given full reign (and full responsibility).
+2. How to work with what you're given, even when extremely sparse.
+3. How to deliver _iteratively (MVP)_.
+4. How to keep parts of the solution decoupled from one another.
+
+&lt;Handout #1 - Morning session&gt;
+
+{:.p1}
+{:style="text-align: center;"}
+**Welcome to the BBC - Part I**
+
+{:.p3}
+Welcome to the BBC - The “Bouncy Ball Company”. You have been selected to join the newly formed Super Hot Automation Facilitation Team, SHAFT for short
+
+{:.p3}
+The focus of this team is to outcompete _every other manufacturer_ by building software that will automate every step in the manufacture of most any sort of ball, starting with the most popular product here in the northeast; the football (American NFL football).
+
+{:.p3}
+Manufacturing overview reference: [http://www.madehow.com/Volume-3/Football.html](http://www.madehow.com/Volume-3/Football.html)
+
+{:.p3}
+For the bill of materials and manufacturing steps, see the link above:
+
+{:.p3}
+Forms teams, possibly based on the stages of the manufacturing process, and be sure to remain agile. More news later
+
+{:.p3}
+The first half of the day (pre-lunch) will be devoted to **_iterating_** and improving the process with at least three deliveries, each accompanied by a customer review and feedback.
+
+&lt;end Handout #1 - Morning session&gt;
+
+{:style="text-align: center;"}
+**LUNCH**
+
+&lt;Handout #2 - Afternoon session&gt;
+
+{:.p1}
+{:style="text-align: center;"}
+**Welcome to the BBC - Part II**
+
+{:.p2}
+Start manufacturing soccer balls. The requirement is to apply all the learning that went into making footballs to now making soccer balls, maximizing the automation by maximizing parameterization, and minimizing waste of the raw materials.
+
+{:.p1}
+Manufacturing overview reference: [http://www.madehow.com/Volume-5/Soccer-Ball.html](http://www.madehow.com/Volume-5/Soccer-Ball.html)
+
+{:.p3}
+As you may have surmised by now, the _real_ exercise is about teamwork, both within teams and _across_ teams, and _rapidly_ finding out what works for the people who show up.
+
+&lt;end Handout #2 - Afternoon session&gt;
+
+Here's what happened:
+
+The morning was predictably defined by lots of questions -
+
+- What constitutes "acceptance"?
+- How will we divide the work up?"
+- How will me track progress?
+
+The answers were much simpler than attendees imagined:
+
+- Ask the customer what is acceptable, and a rank-ordered list of what comes first, second, etc.
+- Organization is entirely up to the team(s).
+- Deliver working (acceptable) software.
+
+The afternoon session raised the bar on agility at both the testing and implementation levels:
+
+- <u>Similarities</u>: Both balls have an inner bladder, are inflated with a given quantity of air, required a valve to inflate the bladder, and require a stitched-together outer skin of  n faces which has to be printed on (e.g. the logo, serial #, etc.)
```

Comment:
We don't have to preserve the original formatting as long as the message stays the same, so I'd rather go with a normal emphasis through `**` than using `<u>`